File: Basic Algorithms/SelectionSort.java
package basicAlgos;

public class SelectionSort {

	public static void main(String[] args) {
		int[] List = new int[] {45,86,95,12,36,47,89,36,98};
		
		SelectionSorting(List);
		
//		for(int i=0; i<List.length; i++) {
//			System.out.println(List[i]);
//		}
		

	}

	public static void SelectionSorting(int[] list) {
				
		for(int i=1; i<list.length; i++) {
			int min_idx = i;
            for (int j = i+1; j < list.length; j++)
                if (list[j] < list[min_idx])
                    min_idx = j;
 
            int temp = list[min_idx];
            list[min_idx] = list[i];
            list[i] = temp;
            System.out.println(list[i]);
		}
				
		
	}

}

File: Basic Algorithms/SelectionSort.js
const data = [5,4,2,3,1,9,25]


const selectionSort = (arr) => {
  const length = arr.length
  for(let i=0; i<length; i++){
    for(let j=0; j<length; j++){
        if(arr[i]<arr[j]) {
           const temp = arr[i];
           arr[i] = arr[j]
           arr[j] = temp;
        }

      }
    }
    return arr
  }
  

selectionSort(data)